src/vs/code/node/cliProcessMain.ts

```diff
@@ -139,8 +139,18 @@ class Main {
 								return TPromise.wrapError(new Error(`${notFound(id)}\n${useId}`));
 							}
 
-							console.log(localize('foundExtension', "Found '{0}' in the marketplace.", id));
-							console.log(localize('installing', "Installing..."));
+							const installedExtension = installed.filter(e => getId(e.manifest) === id)[0];
+							const installedVersion = installedExtension.manifest.version;
+							const newestVersion = extension.version;
+							const shouldUpdate = installedVersion !== newestVersion;
+
```

Comment:
This might also downgrade the extension. You should check if there is a newer version of extension in the market place. For eg [here](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/extensions/node/extensionsWorkbenchService.ts#L190)

Reply:
Done.